Rakefile

```diff
@@ -1,6 +1,10 @@
 require 'rake'
 require 'rake/testtask'
-require 'bundler/gem_tasks'
+
+begin
+  require 'bundler/gem_tasks'
+rescue LoadError
```

Comment:
Why do we use this? We don't care about the gem tasks here?

Reply:
I don't know. They can be used to build and release the gem. I'm not a repo owner so I can't say if it's used or not.

Reply:
It's not :) but if it would we, we should not fail to load them... but it's not important now :shrug: